application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -38,8 +38,8 @@
 public final class VcActivityCommand extends SlashCommandAdapter {
     private static final Logger logger = LoggerFactory.getLogger(VcActivityCommand.class);
 
-    private static final int MIN_AGE_DURATION = 0;
-    private static final long MAX_AGE_DURATION = TimeUnit.DAYS.toSeconds(7);
+    private static final int  MIN_AGE_DURATION_SECONDS = 0;
```

Comment:
You have two spaces now `int__MIN`, also shows me that you pushed without applying spotless - dont.

Reply:
Fixed

Reply:
Its not. You changed completely different parts of the code.

![](https://i.imgur.com/Irwt06I.png)

Reply:
I ran spotless. I will change it manually